workflow/main.py

```diff
@@ -0,0 +1,213 @@
+    print(baseDir)
+    print(tps_in_dir)
+    print(tps_out_dir)
+
+    # * Move files into directory
+    output_sif_orig = os.path.join(baseDir, 'output.sif')
+    activity_tsv_orig = os.path.join(baseDir, 'activity-windows.tsv')
+    output_dest = os.path.join(baseDir, 'workflow', 'output.sif')
+    activity_dest = os.path.join(baseDir, 'workflow', 'activity-windows.tsv')
+
+    os.replace(output_sif_orig, output_dest)
+    os.replace(activity_tsv_orig, activity_dest)
+
+    # input args from command line
+    OUTPUT_FILE = os.path.join(baseDir, 'workflow', 'output.sif')
+    STYLE_FILE = os.path.join(baseDir, 'workflow', 'tps_style.xml')
+
+    # dir name set manually
+    DIRNAME = r"C:\Users\ajshe\OneDrive\Documents\Comp_bio\Cytoscape_v3.7.1"
+    CYTOSCAPE = 'Cytoscape.exe'
+
+    if OUTPUT_FILE.lower().endswith(".sif"):
+        print("file has valid extention")
+    else:
+        print("invalid extension on output file: ", OUTPUT_FILE)
+        sys.exit()
+
+    # get absolute paths if nessesary
+    OUTPUT_FILE = os.path.abspath(OUTPUT_FILE)
+    STYLE_FILE = os.path.abspath(STYLE_FILE)
+    print("---absolute output path: ", OUTPUT_FILE)
+    print("---absolute style path: ", STYLE_FILE)
+
+    # * Gather data used to generate node anotations
+    # Use the version with the header line
+    pepMapFile = os.path.join(tps_in_dir, 'peptide-mapping.tsv')
+    pepFirstFile = os.path.join(tps_in_dir, 'p-values-first.tsv')
+    pepPrevFile = os.path.join(tps_in_dir, 'p-values-prev.tsv')
+    print(pepMapFile)
+    print(pepFirstFile)
+    print(pepPrevFile)
+
+    # Use the version for which log2 fold change has been precomputed
+    timeSeriesFile = os.path.join(
+        baseDir, 'data', 'timeseries', 'log2FoldChange011215.txt')
+    #timeSeriesFile = r"C:\\Users\\ajshe\\Anaconda3\\envs\\py2cyto\\tps\\data\\timeseries\\log2FoldChange011215.txt"
+    print(timeSeriesFile)
+
+    windowsFile = os.path.join(baseDir, 'workflow', 'activity-windows.tsv')
+    networkFile = os.path.join(baseDir, 'workflow', 'output.sif')
+    print(windowsFile)
+    print(networkFile)
+
+    # Use the same EGFR gold standard for Olsen 2006 and Ale's data
+    goldStandardFile = os.path.join(
+        baseDir, 'data', 'resources', 'eight-egfr-reference-all.txt')
+
+    styleTemplateFile = os.path.join(
+        baseDir, 'notebooks', 'tps_style_template.xml')
+
+    out_dir = os.path.join(baseDir, 'results', str(
+        today) + '-and-wolf-yadlin-TPS-cytoscape')
+    if not os.path.exists(out_dir):
+        os.makedirs(out_dir)
+        print('Created {}'.format(out_dir))
+    outFile = os.path.join(
+        out_dir, 'wolf-yadlin-cytoscape-annotations-' + str(today) + '.txt')
+    outStyleFile = os.path.join(out_dir, 'tps_style.xml')
+
+    # * run Utility function
+    pvalThresh = 0.01  # Same threhsold used in TPS
+    logTransform = False
+    pepsPerProt = PrepTemporalCytoscapeTPS(pepMapFile, timeSeriesFile, pepFirstFile,
```

Comment:
If we can log transform the data here, does that eliminate the need to include the log fold change data in the repository?

Reply:
the workflow uses the precomputed log transform file provided. We could generate this file here as well, although I am unfamiliar with how it was computed. What was the methodology used to write this file?